.env.example

```
@@ -17,6 +17,11 @@ SNAPTRADE_USER_SECRET=your_snaptrade_user_secret_here
 # Database
 DATABASE_URL=sqlite:///family_office.db
 
+# SimpleFIN (optional local bank/card sync)
+SIMPLEFIN_SETUP_TOKEN=
+SIMPLEFIN_ACCESS_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SIMPLEFIN_ACCESS_URL key should go before the SIMPLEFIN_SETUP_TOKEN key

(UnorderedKey)